testutil-mentions/src/main/kotlin/io/spine/examples/pingh/testing/mentions/given/PredefinedGitHubSearchResponses.kt

```diff
@@ -150,4 +150,15 @@ public class PredefinedGitHubSearchResponses : GitHubSearch {
     public fun mentionsAreNotFetched() {
         areMentionsFetched = false
     }
+
+    /**
+     * Resets the instance to its initial state.
+     *
+     * Indicates that the service is not frozen and no mentions have been received yet.
```

Comment:
It does not "indicate".

How about this:
```
Once the instance is reset, it becomes no longer frozen, and stores no mentions.
```
Maybe, I am not getting your idea, so please verify my text.